### DISCUSSION TITLE
Remove tracker from a torrent automatically

### DISCUSSION THREAD

**Discussion by lucadaeira68** [rogerfar/rdt-client#622] [Ideas]
Hello,

I'd just like to suggest an improvement for the RDT client. With the latest updates, it's no longer possible to add torrents from YGG, but if you delete the tracker and re-add the torrent in RD, the torrent goes through without a problem. Can we add an option to automatically delete the tracker before sending it to RD?

**Comment by rogerfar** [comment]
Sorry but this doesn't make much sense.. Delete the tracker from where?

**Reply by lucadaeira68** [reply]
From the torrent. 

Let me explain: 

Radarr selects a torrent and sends it to RDT-client. Then it appears on Real Debird etc.

What I'd like is for Radarr to select the torrent, send it to RDT-client, which will remove the tracker from the torrent by editing the torrent (it's possible to do this manually with tools or even with a script, I have a python script that does this) and then send it to RD. This avoids the Infringin torrent error.

**Reply by rogerfar** [reply]
Sorry for the late reply, but does it avoid the infringement errors? But if it has 0 trackers and they don't have the torrent cached yet, how will it download?
